packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -277,6 +277,41 @@ correct log retention period (never expire, by default).
 *Further note* that, if the log group already exists and the `logRetention` is not set, the custom resource will reset
 the log retention to never expire even if it was configured with a different value.
 
+### FileSystem Access
+
+You can configure a function to mount an Amazon Elastic File System (Amazon EFS) to a
+directory in your runtime environment with the `filesystems` property. To access Amaozn EFS
+from lambda function, the Amazon EFS access point will be required. 
+
+Amazon EFS _access points_ are application-specific entry points into an EFS file system that
+make it easier to manage application access to shared datasets. Access points can enforce a user
+identity, including the user's POSIX groups, for all file system requests that are made through
+the access point. Access points can also enforce a different root directory for the file system
+so that clients can only access data in the specified directory or its subdirectories. See
+[Working with Amazon EFS Access Points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html) for more details.
+
+The following sample enables the lambda function to mount the Amazon EFS filesystem via the `accessPoint` to `/mnt/msg` in the runtime environment.
+
+```ts
+const accessPoint = new efs.AccessPoint(stack, 'AccessPoint', {
+  fileSystem,
+});
```

Comment:
There should be a `fileSystem.addAccessPoint()` method

Reply:
Agreed.

FWIW - we agreed this will come as a separate PR - https://github.com/aws/aws-cdk/pull/8631#discussion_r443411910.

Reply:
Yes. PR created https://github.com/aws/aws-cdk/pull/8737